ansible/roles/shared_dir.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+- name: SharedDirHost
+  hosts: shared_dir_host  # Should be one
+  sudo: yes
+  vars_files:
```

Comment:
FYI we can use this pattern to share variables between two roles and not put them in the global variable file